core/trino-main/src/main/java/io/trino/operator/aggregation/partial/SkipAggregationBuilder.java

```diff
@@ -16,25 +16,15 @@
 import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
I think this is a wrong package. Other builders are in `builder` package. I don't think we need `partial` package